packages/playwright-core/src/server/frames.ts

```diff
@@ -1297,9 +1297,9 @@ export class Frame extends SdkObject {
       omitAttached = true;
     else if (options.isNot && options.expression === 'to.have.count' && options.expectedNumber !== 0)
       omitAttached = true;
-    else if (!options.isNot && options.expression.endsWith('.array') && options.expectedText!.length === 0)
+    else if (!options.isNot && options.expression.endsWith('.array') && options.expectedText?.length === 0)
```

Comment:
Why did this change?

Reply:
Because expectedText could be undefined, and it is in my case.